Life as a teacher (#4120)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher.md

---
title: "Life as a teacher"

> description: Deciding if teaching is right for you? Learn how rewarding life as a teacher can be, including pay, benefits, and career progression.

Author: ```suggestion
description: Exploring if teaching is the right career for your? Learn how rewarding life as a teacher can be, including pay, benefits, and career progression.
```
@Sarah-DfE how about this?

Reviewer: Like it!

> description: Wondering whether teaching is the right career for your? Learn how rewarding life as a teacher can be, including pay, benefits, and career progression.

Author: I agree! I think we will need to revist/check these as we introduce the new pages as there may be overlap.

content:
image: "static/images/content/hero-images/0029.jpg"
colour: "yellow"
layout: "layouts/category"
title_paragraph: |-

>     Choosing or changing a career can be a big decision. Find out what it's like to be a teacher in England, including how varied and rewarding teaching can be.

Reviewer: Do we need to mention life as a teacher here, just thinking for consistency, could we have 'Find out what life is like as a teacher in England' etc, just a thought!

>     Choosing or changing a career can be a big decision. Find out what it's like to be a teacher in England, including how varied and rewarding teaching can be.

Author: ```suggestion
    Choosing or changing a career can be a big decision. Find out what life is like as a teacher in England, including how varied and rewarding teaching can be.
```
